lms/envs/production.py

```diff
@@ -1078,9 +1078,15 @@ def should_show_debug_toolbar(request):
 if LRS_MONGO_HOST:
     LRS_MONGO_CLIENT = pymongo.MongoClient(host=LRS_MONGO_HOST, readPreference='secondaryPreferred')
     LRS_MONGO_DB = LRS_MONGO_CLIENT[AUTH_TOKENS.get('LRS_MONGO_DB', 'dataproduct')]
+    LRS_INTEGRATIONS_MONGO_DB = LRS_MONGO_CLIENT[AUTH_TOKENS.get('LRS_INTEGRATIONS_MONGO_DB', 'integrations')]
```

Comment:
this is a funny one - we might want two different mongo users (and two mongo_host settings) - one with read access to the dataproduct for the live data and another for writing to the integration collection. I usually have them split at the datamesh end.
Alternately we might want different settings files for the EDX django server vs the worker

Reply:
Which one would you prefer?

Reply:
I kinda like the idea of the worker having very specific access